scripts/curate_data_axondeepseg_tem.py

```diff
@@ -145,8 +145,17 @@ def main(root_data, output_data):
         json.dump(data_json, json_file, indent=4)
 
     # Create README
+    readme_text ="""- TEM dataset for AxonDeepSeg (https://axondeepseg.readthedocs.io/) \n
+- 158 brain (splenium) samples from 20 mice with axon and myelin manual segmentation labels. \n
+- 20160718_nyu_mouse_25_0002 was omitted because it contained incomplete data in the folder /duke/projects/axondeepseg/raw_data/data_TEM/3_done/ \n
+- Original source files are located in duke/histology/mouse/20160718_nyu_mouse \n
+- Our original paper (Aldo et al. 2018), the FOV was reported to be 6x9 um^2. This is because 1) these original values were reported in the original data reference (below), and 2) our images here are slightly cropped at the bottom relative to the original data in order to remove the scale bar. \n
```

Comment:
Aldo --> Zaimi

Reply:
```suggestion
- Our original paper (Zaimi et al. 2018), the FOV was reported to be 6x9 um^2. This is because 1) these original values were reported in the original data reference (below), and 2) our images here are slightly cropped at the bottom relative to the original data in order to remove the scale bar. \n
```
Oups! It was written as `Aldo et al.` since forever in the TEM dataset on OSF 😬.